examples/single/native/android/app/build.gradle

```diff
@@ -128,7 +128,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.solidauthfetcherreactnativedemo"
+        applicationId "com.solidClientAuthnreactnativedemo"
```

Comment:
Casing here and elsewhere looks weird - I'd expect `solidClientAuthnReactNativeDemo`

Reply:
(Which you can ignore if you're hiding the native examples for now anyways)

Reply:
The casing is fixed in 7351f11ba1e3b6b02dafd295d24a6e2984138d80, but yeah I'll remove the native examples from the docs anyway, because that's not a conventional Java package name.